python/ray/tests/test_logging_2.py

```diff
@@ -207,6 +207,51 @@ def test_text_mode_actor(self, shutdown_only):
     logging_config=ray.LoggingConfig(encoding="TEXT")
 )
 
+@ray.remote
+class actor:
+    def __init__(self):
+        pass
+
+    def print_message(self):
+        logger = logging.getLogger(__name__)
+        logger.info("This is a Ray actor")
+
+actor_instance = actor.remote()
+ray.get(actor_instance.print_message.remote())
+"""
+        stderr = run_string_as_driver(script)
+        should_exist = [
+            "job_id",
+            "worker_id",
+            "node_id",
+            "actor_id",
+            "task_id",
+            "INFO",
+            "This is a Ray actor",
+        ]
+        for s in should_exist:
+            assert s in stderr
+
+    @pytest.mark.parametrize(
+        "ray_start_cluster_head_with_env_vars",
+        [
+            {
+                "env_vars": {
+                    "RAY_SETUP_DEFAULT_LOGGER": "1",
```

Comment:
The diff looks kinda odd, but this is pretty much the same test except this evn is set here and that one line 254 `ray.init` isn't called with logging config